pom.xml

```diff
@@ -45,12 +45,13 @@
 
     <!-- matching armeria/grpc/zipkin -->
     <spring-boot.version>2.1.4.RELEASE</spring-boot.version>
-    <armeria.version>0.83.0</armeria.version>
+    <armeria.version>0.85.0</armeria.version>
```

Comment:
note this is later than last released zipkin. we'll probably not want to release this until they are synced (likely 0.86)